[![GitHub version](https://img.shields.io/badge/version-1.0.0-blue.svg)](https://github.com/AMRverse/AMRrulebrowser)
[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
<!-- [![DOI](https://zenodo.org/badge/615052960.svg)](https://zenodo.org/doi/10.5281/zenodo.10810218) -->

<img src="logo.png" width="150" align="left">


<p align="left">
  <em><b>Web browser to interactively explore AMRrules</b></em>
</p>

This web browser was developed for the interactive exploration of [AMRrules](https://github.com/AMRverse/AMRrules). The AMRrules Python package contains the core rule set alongside tools to interpret AMR genotypesâ€”currently optimized for [AMRfinderplus](https://www.ncbi.nlm.nih.gov/pathogens/antimicrobial-resistance/AMRFinder/), output. By applying these rules, the system generates comprehensive genome reports that translate complex genetic data into expert-level insights on how core genes, acquired genes, and mutations influence antimicrobial susceptibility.

Full specification: [AMRrules spec v0.6](https://docs.google.com/spreadsheets/d/1F-J-_8Kyo3W0Oh6eDYyd0N8ahqVwiddM2112-Fg1gKc/edit?usp=sharing). Note this includes several additional fields beyond those pictured above, including NCBI and CARD ARO accessions to uniquely identify genes; details of the breakpoints and standards used; evidence codes, grades and limitations; and a rule annotation note.

### Available rules

Rule curation is a work in progress, under active development by the [ESGEM-AMR](https://github.com/AMRverse/AMRrulesCuration/) Working Group.
